site/src/@ndlib/gatsby-theme-marble/assets/i18n/json/common.en.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Overrides for text will go here and in the `text.en.json` file. This will be for cases where we are making specific mentions of the University or other text the is "site" specific.